src/widget.ts

```diff
@@ -51,11 +56,64 @@ export class DeephavenModel extends DOMWidgetModel {
 export class DeephavenView extends DOMWidgetView {
   private iframe: HTMLIFrameElement;
 
+  sendAuthenticationResponse = (
+    messageId: string,
+    source: WindowProxy,
```

Comment:
In the context of this function, `source` doesn't really make sense. `source` is the source of the event, but in the context of this function, we're just passing a function to a child window. I'd suggest changing the parameter name to `childWindow` here (or something similar).